### ISSUE TITLE
How to deploy on Heroku

### ISSUE THREAD

**Issue by scollovati** [blackboard/BBDN-LTI-Tool-Provider-Node#127]
Dear BB Devs,
this is a great tool for testing LTI integrations. Do you mind in creating instructions for deploying it to Heroku?

I've tried but w/o success (I'm not a NodeJS expert). The main issue may be that the Heroku's Dyno process port is randomly generated, while here we have to declare it in the server config.

**Comment by ShannonH** [comment]
I'm working on some wiki pages, I started with basic Heroku instructions [here](https://github.com/blackboard/BBDN-LTI-Tool-Provider-Node/wiki/How-to-deploy-to-various-cloud-platforms)

**Comment by scollovati** [comment]
Hi @ShannonH thanks for this. I've tried the basic instructions but I've encountered a couple of issues:
1. I had to change the url here in order to match with the heroku one https://github.com/scollovati/BBDN-LTI-Tool-Provider-Node/blob/master/server/config/config.json#L2 . The instructions written here seem to be no more relevant: https://github.com/scollovati/BBDN-LTI-Tool-Provider-Node/blob/master/README.md?plain=1#L15-L24
2. It's not clear how to register the platform inside this tool. There is no field for putting the deploymentId and it's not clear which developer portal we should choose:
![immagine](https://user-images.githubusercontent.com/20740642/156889876-280bd23e-218c-4bf5-829c-86558c4b0019.png)
3. There is no persistence for the platforms configured, hence it is impossible to test it with Blackboard Learn.
